### PR TITLE
[BUG] Make source map not use grammars-Total

### LINKED ISSUE #4367: [BUG] Sourcemap Dependent on grammars-Total
The source map is dependent on the merged grammars in **grammars-Total**. 
I thought that these were also generated after deployment, but I was wrong 😅. 

This causes the skipping faulty #4320 to not work the very first time the level is executed which is undesired (I think?).
(btw, I found this bug by testing skipping faulty on alpha)

### PR DESCRIPTION
**Pull request by ToniSkulj**
Fixes #4367

### CODE REVIEW

**Comment by ToniSkulj** [comment]
Hey @Felienne! Do you think we can merge this when you have the time? Alpha is still bugged out after merging #4320  (sorry for that), this PR fixes that

afbeelding

**Review by Felienne — APPROVED** [review]
Ah sorry, here you go! Let me know if this fixes the issues on Alpha!

**Comment by ToniSkulj** [comment]
> Ah sorry, here you go! Let me know if this fixes the issues on Alpha!

Thank you, will do!

**Pull request merged**
